Resources/migration.yml

```diff
@@ -537,3 +537,8 @@ LightTree03: LightTree
 LightTree04: LightTree
 LightTree05: LightTree
 LightTree06: LightTree
+
+#2024-12-28
+DrinkIrishCarBomb: DrinkIrishSlammer
+IrishCarBomb: IrishSlammer
+irishcarbomb: irishslammer
```

Comment:
only entities
```suggestion
DrinkIrishCarBomb: DrinkIrishSlammer

```

Reply:
Thanks, I was just adding anything with an id under the prototypes folder, like a rube. Commited your correction.

Reply:
Somehow go prototype and ID stuck in my head, instead of entities.